lib/gateway/Shard.js

```diff
@@ -492,7 +497,7 @@ class Shard extends EventEmitter {
                 /**
                 * Fired when a message is updated
                 * @event Client#messageUpdate
-                * @prop {Message} message The updated message. If oldMessage is null, it is recommended to discard this event, since the message data will be very incomplete (only `id` and `channel` are guaranteed)
+                * @prop {Message} message The updated message. If oldMessage is null, it is recommended to discard this event, since the message data will be very incomplete (only `id` and `channel` are guaranteed). If the channel isn't cached, `channel` will be an object with only `id`.
```

Comment:
consistency
```suggestion
                * @prop {Message} message The updated message. If oldMessage is null, it is recommended to discard this event, since the message data will be very incomplete (only `id` and `channel` are guaranteed). If the channel isn't cached, `channel` will be an object with an `id` key.
```